run-test.sh

```diff
@@ -120,7 +119,7 @@ test_example () (
   cp -r "$examplePath" ./example
   pushd ./example
   prepare_shell
-  nix run .\#regenerate-files --print-build-logs
+  # nix run .\#regenerate-files --print-build-logs
```

Comment:
Is this line still there on purpose or should it just be scraped entirely?

Reply:
Nope, that's purely accidental. Removed it